devito/operator/operator.py

```diff
@@ -1300,7 +1300,11 @@ def nbytes_avail_mapper(self):
                     continue
                 try:
                     if i._mem_mapped:
-                        mapper[device_layer] -= i.nbytes
+                        try:
+                            v = self[i.name]._obj.nbytes
```

Comment:
just self.get(i.name, i) no? WHy would i.nvytes work but self[i.name] need _obj?

Reply:
Another stupid question,  how can we end up in the except? At this point all arguments should be resolved if it's not in `self` we're missing an argument no?

Reply:
the EnrichedTimeFunction's may have not been resolved at this point yet